src/pivot-table/hooks/use-column-width.ts

```diff
@@ -67,12 +69,16 @@ export default function useColumnWidth(
   const styleService = useStyleContext();
   const { measureText: measureTextForHeader } = useMeasureText({
     ...styleService.header,
-    bold: true,
+    bold: isBold(styleService.header),
+  });
+  const { estimateWidth: estimateWidthForMeasureValue } = useMeasureText({
```

Comment:
TODO: Update `useMeasureText` to except `fontWeight` as a property instead. This would be a "breaking change" in the utils repo and require updates in both tables repos.

Reply:
btw, this one should wait for `useMeasureText` update from shared lib, right?

Reply:
No. I don't think there is a reason to wait. The currently logic works just fine. But we probably want to add the option to set font weight through the theme. Then this logic will not work as the user can set other values than `600/normal/undefined`.

I said probably here as well since the interaction between the `font-weight` value from the theme and the styling panels "bold" button is going to be tricky at best. It would also include dealing with custom `font-style` and `text-decoration`.